recipe/build.sh

```diff
@@ -1,102 +1,44 @@
-#  - non-(minor)-version specific files go in PREFIX/lib/perl5
-#  - (minor) version specific files go in, e.g., PREFIX/lib/perl5/5.32
-
-perl_lib=".../../lib/perl5"
-perl_archlib="${perl_lib}/${PKG_VERSION%.*}"
-perl_core=/core_perl
-perl_site=/site_perl
-perl_vendor=/vendor_perl
-
-_config_args+=(
-  "-Dprefix=${PREFIX}"
-  "-Dvendorprefix=${PREFIX}"
-
-  "-Dscriptdir=${PREFIX}/bin"
-  "-Dsitescript=${PREFIX}/bin"
-  "-Dvendorscript=${PREFIX}/bin"
-
-  -Duserelocatableinc
-  -Duseshrplib
-  -Dinc_version_list=none
-
-  "-Dprivlib=${perl_lib}${perl_core}"
-  "-Dsitelib=${perl_lib}${perl_site}"
-  "-Dvendorlib=${perl_lib}${perl_vendor}"
-
-  "-Darchlib=${perl_archlib}${perl_core}"
-  "-Dsitearch=${perl_archlib}${perl_site}"
-  "-Dvendorarch=${perl_archlib}${perl_vendor}"
-)
-
-_config_args+=(-Dinstallusrbinperl=n)
-
+_config_args+=(-Dprefix="${PREFIX}")
 _config_args+=(-Dusethreads)
-_config_args+=(-Dcccdlflags="-fPIC")
-_config_args+=(-Dldflags="${LDFLAGS} ${archflags}")
+_config_args+=(-Duserelocatableinc)
+_config_args+=(-Dcccdlflags="-fPIC ${CFLAGS}")
+_config_args+=(-Dldflags="${LDFLAGS}")
 # .. ran into too many problems with '.' not being on @INC:
 _config_args+=(-Ddefault_inc_excludes_dot=n)
-
-if [[ -n "${ccflags}" ]]; then
-  _config_args+=(-Dccflags="${ccflags}")
-fi
 if [[ -n "${GCC:-${CC}}" ]]; then
   _config_args+=("-Dcc=${GCC:-${CC}}")
 fi
-if [[ -n "${AR}" ]]; then
-  _config_args+=("-Dar=${AR}")
-fi
-if [[ "${target_platform}" == linux-* ]]; then
+if [[ ${HOST} =~ .*linux.* ]]; then
   _config_args+=(-Dlddlflags="-shared ${LDFLAGS}")
-elif [[ "${target_platform}" == osx-* ]]; then
-  _config_args+=(-Dsed=/usr/bin/sed)
+# elif [[ ${HOST} =~ .*darwin.* ]]; then
 #   _config_args+=(-Dlddlflags=" -bundle -undefined dynamic_lookup ${LDFLAGS}")
 fi
 # -Dsysroot prevents Configure rummaging around in /usr and
 # linking to system libraries (like GDBM, which is GPL). An
 # alternative is to pass -Dusecrosscompile but that prevents
 # all Configure/run checks which we also do not want.
-# -Dsysroot prevents Configure rummaging around in /usr and
-# linking to system libraries (like GDBM, which is GPL). An
-# alternative is to pass -Dusecrosscompile but that prevents
-# all Configure/run checks which we also do not want.
-_config_args+=("-Dsysroot=${CONDA_BUILD_SYSROOT}")
-
-_config_args+=(
-  -Dmyhostname=conda
-  -Dmydomain=.conda
-  -Dperladmin=conda
-  -Dcf_by=conda
-  -Dcf_email=conda
-)
-
-_config_args+=(
-  "-Dsysman=${PREFIX}/man/man1"
-  "-Dman1dir=.../../man/man1"
-  "-Dman3dir=.../../man/man3"
-)
+if [[ -n ${CONDA_BUILD_SYSROOT} ]]; then
+  _config_args+=("-Dsysroot=${CONDA_BUILD_SYSROOT}")
+else
+  if [[ -n ${HOST} ]] && [[ -n ${CC} ]]; then
+    _config_args+=("-Dsysroot=$(dirname $(dirname ${CC}))/$(${CC} -dumpmachine)/sysroot")
+  else
+    _config_args+=("-Dsysroot=${CONDA_BUILD_SYSROOT}/usr")
+  fi
+fi
 
-./Configure -de "${_config_args[@]}"
+./Configure "${_config_args[@]}" \
+                -de
 make
```

Comment:
Can we use `make -j${CPU_COUNT}`?

Reply:
hmm, I try to avoid the -j option, as it might lead to too many threads running .... eg on ppc64le

Reply:
I use the build-number 1 by intention, as there is already for osx-arm64 the build-number 0 available.

Reply:
you could find out by either looking on zeus directly, or by checking via 'CONDA_SUBDIR=osx-arm64 conda search perl=5.34'